Makefile

```diff
@@ -13,6 +13,10 @@ bin-race:
 	./scripts/build_tools.sh -race
 docker:
 	./scripts/build_docker.sh
+dev:
+	./scripts/build.sh
+	./scripts/build_docker.sh
```

Comment:
it is very rare that you have to rebuild the docker image when using the docker-dev stack. the whole point of the docker-dev stack is that it'll mount your custom binary and configs without having to build an image.

also you're better off using the launch.sh script (since it sets up a bunch of stuff up by default) instead of calling docker-compose directly.

I recommend taking this commit out of this PR, i would accept it as a new pr with the changes described